CMakeLists.txt

```diff
@@ -146,7 +122,7 @@ if(CAMODOCAL_PLATFORM_UNIX)
     #set_linker_flags("-Wl,--no-undefined -ldl -lnsl -lm -pthread -lrt" CACHE)
     find_package(LibDL)
     find_package(Libnsl)
-    find_package(LibM)
+    # find_package(LibM)
```

Comment:
Good question. What's LibM and what is the difference between Libm and LibM? I have no idea. Libm break my compilation and removing it seems not produce any effects.